addons/phone_validation/tests/test_phonenumbers_patch.py

```diff
@@ -30,3 +30,17 @@ def test_region_CI_monkey_patch(self):
         else:
             self.assertFalse(hasattr(phonenumbers_patch, '_local_load_region'),
                 "The code should not be monkey patched with phonenumbers > 8.12.32.")
+
+    def test_region_BR_monkey_patch(self):
```

Comment:
Maybe specify in doctstring that it tests a patch for the added number in mobile phones, easier than blaming

Reply:
Done